lib/operations/logic/truthy.js

```diff
@@ -0,0 +1,3 @@
+import truthy from '../../helpers/truthy';
+truthy.code = '!!';
```

Comment:
Same here `code` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014

`!` is an existing operator